jekyll/_cci2/runner-installation.adoc

```diff
@@ -9,105 +9,38 @@ version:
 :toc: macro
 :toc-title:
 
-NOTE: CircleCI runner is currently only available if you are on the https://circleci.com/pricing[Scale Plan]. Please reach out to your sales representative for information on how to sign up for the Scale Plan.
+NOTE: CircleCI runner is currently available on the https://circleci.com/pricing[Scale Plan]. Please reach out to your sales representative for information on how to sign up for the Scale Plan.
```

Comment:
same comment from the other change:

Does it make sense to push interested readers to reach out if they don't have a sales rep: "Please reach to your sales representative or contact us <linked to https://circleci.com/contact-us/?cloud> for information on ..."?